site/src/components/gradient-card.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Gradient card
+description: The gradient card component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - gradient-card
+---
+
+# Gradient card
+
+The `.gradient-card` and `.gradient-card-content` classes contain a predefined gradient border, default border radius and padding. Gradient cards are used to bring attention to the card content.
```

Comment:
What about this?

```
Gradient card component displays content and actions on a single topic.
```

We need to have an explanation of what the component was created for, and how to use it. Describing its visual properties is not very helpful. But I'm struggling with describing how it's different from the card component 😞 

@BenDMyers @homing1 please help with the wording here 🙏🏻

Reply:
In my mind, the gradient card is very similar to the card component. The nature of gradient is indeed a visual distinction though. Otherwise, you would just use the card component. The other difference is that the card component provides slightly more structure.

Would something like this make sense? "Gradient card component is similar to the card component, in which it is used for representing data and actions. Use the gradient card component to add a predefined gradient border to provide extra emphasis and draw extra attention to the card content."

Reply:
Works for me, updated.